/*
 * Copyright 2018 The Feast Authors
 *
 * Licensed under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License.
 * You may obtain a copy of the License at
 *
 *     https://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software
 * distributed under the License is distributed on an "AS IS" BASIS,
 * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 * See the License for the specific language governing permissions and
 * limitations under the License.
 */

syntax = "proto3";

import "google/protobuf/timestamp.proto";
import "feast/types/Feature.proto";

package feast.types;

option java_package = "feast.types";
option java_outer_classname = "FeatureRowProto";
option go_package = "github.com/gojek/feast/protos/generated/go/feast/types";

message FeatureRow {
  string entityKey = 1;
  repeated Feature features = 2;
  google.protobuf.Timestamp eventTimestamp = 3;
  string entityName = 4;
}
